pkgs/tools/misc/kamp/package.nix

```
@@ -0,0 +1,39 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  replaceVars,
+  perl,
+  stdenv,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "kamp";
+  version = "0.2.1";
+
+  src = fetchFromGitHub {
+    owner = "vbauerster";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-9cakFhA9niMZ0jD0ilgCUztk4uL6wDp6zfHUJY/yLYw=";
```

```suggestion
    hash = "sha256-9cakFhA9niMZ0jD0ilgCUztk4uL6wDp6zfHUJY/yLYw=";
```

Prefer `hash` to `sha256`